File: backend/src/main/java/com/kh/backend/domain/user/model/vo/Memo.java
package com.kh.backend.domain.user.model.vo;

import java.sql.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Memo {

	private int memoNo;
	private int userNo;
	private String content;
	private String targetDate; // 일단 String
}

File: service-frontend/src/type/memo.ts
export interface Memo {
    memoNo: number,
    userNo: number,
    content: string,
    targetDate: string
}

export const initialMemo: Memo = {
    memoNo: 0,
    userNo: 0,
    content: '',
    targetDate: ''
}

export const initialMemoList: Memo[] = [];